common/recommender/Baseline.js

```diff
@@ -2,6 +2,7 @@
 
 const URL = require("../../lib/vendor.bundle.js").urlParse;
 const {INFINITE_SCROLL_THRESHOLD} = require("../constants");
+const COEFFICIENTS = [0.4, 0.7, 0.1, -0.4, -0.2, -0.1];
```

Comment:
This seems to be the same values as in [package.json](https://github.com/mozilla/activity-stream/pull/1106/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R185) edits below:

```js
 +    {
 +      "name": "weightedHighlightsCoefficients",
 +      "title": "recommender weights",
 +      "description": "weightedHighlightsCoefficients",
 +      "type": "string",
 +      "value": "[0.4, 0.7, 0.1, -0.4, -0.2, -0.1]",
 +      "hidden": true
      }
```

Not sure if there is a way to consolidate the values so they don't get out of sync if somebody tweaks these values in the future.